Makefile

```diff
@@ -632,12 +632,12 @@ cc-check: $(TESTS:.cc=)
 	exit "$$_ec";
 
 #: Run all bats tests
+BATS_FILE ?= $(MAKEFILE_DIR)/tests
```

Comment:
Question : what's the extra variable for?

Reply:
You can override this variable at the command line to run just a single test file.